### PR TITLE
Social sources article

### PR DESCRIPTION
**Pull request by ivanklimuk**

### CODE REVIEW

File: src/social-volume-metrics/index.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+The **Spam Index** is the ratio between the **Social Volume** and the **Unique Social Volume**. If the social data didn't contain any duplicated texts, the spam index would be always equal to 1. In fact it's not and the number indicates how many of the documents in the social volume are duplicates.
+
+### Measuring Unit
+
+Relative number, greater or equal then 1.
+
+### Frequency
+
+Same as **Social Volume**.
```
Comment (ipranovich) [inline src/social-volume-metrics/index.md]: Looks great! 
An idea: if here, for every "Same as" case can be added link to same header in social volume, what do you think?

File: src/social-volume-metrics/index.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+title: Social volume metrics
+author: Ivan Klimuk
+date: 2019-09-19
+---
```
Comment (tzanko-matev) [inline src/social-volume-metrics/index.yaml]: Can you add references to where in the code these metrics are computed. Change in which parts of the code should lead to change of this article?

File: src/social-volume-metrics/index.md (hunk 1 above)
Comment (ivanklimuk) [reply src/social-volume-metrics/index.md]: yeah sounds reasonable

File: src/social-volume-metrics/index.yaml (hunk 2 above)
Comment (ivanklimuk) [reply src/social-volume-metrics/index.yaml]: that's the tricky part - we don't really compute these metrics, they are fetched on the fly. So we should refer to ES, and the API's built on top of it

File: src/social-volume-metrics/index.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+### Latency
+
+The latency of **Social Volume** depends on the latency of the documents from each [data source](social-data/). Real-time social data means that the volume calculated on top of this data is also real-time (max. 1-2 seconds).
+
+### Available Assets
+
+We do not separate or filter the [social data](social-data/) being collected by assets. I.e. we can have the social volume for any asset.
+
+By default we use an auto-generated seqrch query to track the volume of mentions for each project, e.g.: `btc OR bitcoin NOT cash NOT gold NOT abc ...etc`. These queries can be manually modified by our admins to exclude redundant results or include some additional word to the query if necessary.
```
Comment (ipranovich) [inline src/social-volume-metrics/index.md]: seqrch -> search

File: src/social-volume-metrics/index.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+
+## 3. Spam Index
+
+### Definition
+
+The **Spam Index** is the ratio between the **Social Volume** and the **Unique Social Volume**. If the social data didn't contain any duplicated texts, the spam index would be always equal to 1. In fact it's not and the number indicates how many of the documents in the social volume are duplicates.
+
+### Measuring Unit
+
+Relative number, greater or equal then 1.
```
Comment (ipranovich) [inline src/social-volume-metrics/index.md]: then -> than, it it's applicable here in general...
Double check with Kim?

File: src/social-volume-metrics/index.md [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+|Sanbase|:x:|:x:|:x:|:x:|:x:|
+|SANGraphs|:x:|:x:|:white_check_mark:|:white_check_mark:|:white_check_mark:|
+|Neuro|:x:|:x:|:x:|:x:|:x:|
+|SANSheets|:x:|:x:|:x:|:x:|:x:|
+
+## 4. Relative Social Dominance
+
+### Definition
+
+The **Relative Social Dominane** (or Social Dominance) for a given asset is the ratio of the **Social Volume** of this asset in comparison to the sum of social volumes for a set of selected assets (by default it's top 50 by market capitalization). E.g. when on a given day the **Social Dominance** of `Ethereum` reaches 23%, this can be explained as _"From all the messages that mentioned any of the top 50 assets, 23% mentioned Ethereum"_.
```
Comment (ipranovich) [inline src/social-volume-metrics/index.md]: Dominane -> Dominance

File: src/social-volume-metrics/index.md (hunk 4 above)
Comment (ivanklimuk) [reply src/social-volume-metrics/index.md]: no no that's totally true - it's a typo

**Pull request merged**